<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhLowercaseOnly</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="patient_list_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>551</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_textbox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search by Name or Phone Number...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_patient_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View Patient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_search_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_New_Patient"/>
    <addaction name="actionDelete_Patient"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionCreate_New_Patient">
   <property name="text">
    <string>Create New Patient</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDelete_Patient">
   <property name="text">
    <string>Delete Patient</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
